src/avitab/apps/MapApp.cpp

```diff
@@ -488,6 +489,38 @@ void MapApp::startCalibrationStep1() {
     keyboard->alignInBottomCenter();
 }
 
+double MapApp::getCoordinate(std::string coordStr) {
+    // Validate characters, since may use physical keyboard to input, and someone will
+    // mistype or attempt to enter DMS using ` (backtick) or " as a separator.
+    std::size_t found = coordStr.find_first_not_of("+-1234567890.,' ");
+    if (found != std::string::npos)  {
+        LOG_ERROR("Error parsing calibration coordinate %s, invalid character %c", coordStr.c_str(), coordStr[found]);
+        return std::numeric_limits<float>::quiet_NaN();

```

Comment:
Shouldn't this throw an exception instead of returning NaN? The calling function even has an exception handler already.

Reply:
OK will do